v5.4.1 (#30959)
mui/material-ui: CHANGELOG.md

# [Versions](https://mui.com/versions/)

## 5.4.1

<!-- generated comparing v5.4.0..master -->

> _Feb 7, 2022_

Reviewer: ```suggestion
_Feb 8, 2022_
```

Author: Now I know why I should complete the release within the day 😂, so that I don't miss the date.

A big thanks to the 24 contributors who made this release possible. Here are some highlights ✨:

> - ♿️ Ensured Snackbar messages are announced by screen reader (#30774) @eps1lon

Reviewer 2: ```suggestion
- ♿️ Snackbar messages are now announced by screen readers (#30774) @eps1lon
```

Reviewer 3: It sounds like they weren't previously with your suggestion. 

We just fixed it for nvda+firefox. The other combinations were working as far as we know.

Though I don't know how to best describe this without making it sound boring 😆

Reviewer 2: Oh, sorry. Thanks for the heads up! Updated it to be more accurate, let me know if it's correct.

> - Number of 🐛 bug fixes and 📚 documentation improvements.

Reviewer 2: ```suggestion
- Several 🐛 bug fixes and 📚 documentation improvements.
```

### `@mui/material@5.4.1`

- &#8203;<!-- 37 -->[AvatarGroup] Enable targeting of additional Avatar when max props is passed (#30794) @mogrady88

> - &#8203;<!-- 36 -->[Badge] fix showzero and invisible condition (#30899) @alisasanib

Reviewer 4: ```suggestion
- &#8203;<!-- 36 -->[Badge] Fix showzero and invisible condition (#30899) @alisasanib
```

- &#8203;<!-- 35 -->[ButtonBase] Expose ref to TouchRipple (#30901) @m4theushw

> - &#8203;<!-- 15 -->[Fab] supporting other colors (#30846) @alisasanib

Reviewer 4: ```suggestion
- &#8203;<!-- 15 -->[Fab] Add support for the default theme colors (#30846) @alisasanib
```

- &#8203;<!-- 09 -->[Snackbar] Ensure messages are announced in NVDA+FF (#30774) @eps1lon

### `@mui/base@5.0.0-alpha.68`

- &#8203;<!-- 10 -->[SelectUnstyled] Improve exported types (#30895) @michaldudak

### `@mui/lab@5.0.0-alpha.68`

- &#8203;<!-- 12 -->[Pickers] Fix `onDismiss` handler in `MobileDatePicker` (#30768) @Ashish2097
- &#8203;<!-- 06 -->[TimePicker] Add font family for clock numbers (#30738) @alisasanib

### `@mui/joy@5.0.0-alpha.14`

- &#8203;<!-- 14 -->[Joy] Add `IconButton` component (#30864) @siriwatknp
- &#8203;<!-- 13 -->[Joy] Use icon inside a Button (#30803) @siriwatknp

### Docs

- &#8203;<!-- 16 -->[examples] Fix vitejs example and improve HMR (#30897) @mihailgaberov
- &#8203;<!-- 38 -->[docs] Improve description of the disableRestoreFocus prop of the `TrapFocus` (#30912) @flaviendelangle
- &#8203;<!-- 32 -->[docs] Remove ul with div children and replace with nav element (#30534) @joeframbach
- &#8203;<!-- 31 -->[docs] Add Saleor to showcase (#30924) @cherniavskii
- &#8203;<!-- 30 -->[docs] Include JSS in styling solution interoperability guide (#30736) @garronej

> - &#8203;<!-- 29 -->[docs] fix contents of link-underline-hover (#30904) @pppp606

Reviewer 4: ```suggestion
- &#8203;<!-- 29 -->[docs] Fix contents of link-underline-hover (#30904) @pppp606
```
